docs/framework/unmanaged-api/debugging/ixclrdataprocess-interface.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "IXCLRDataProcess Interface"
+ms.date: "01/15/2019"
+api.name:
+  - "IXCLRDataProcess"
+api.location:
+  - "mscordacwks.dll"
+api.type:
+  - "COM"
+f1.keywords:
+  - "IXCLRDataProcess Interface"
+helpviewer.keywords:
+  - "IXCLRDataProcess Interface [.NET Framework debugging]"
+topic_type:
+  - "apiref"
+author: "cshung"
+ms.author: "andrewau"
+---
+# IXCLRDataProcess Interface
+Provide methods for querying information about a process.
+
+## Methods
+|Method                           |Description                                                         |
+|---------------------------------|--------------------------------------------------------------------|
```

Comment:
The member names should link to the member documentation.
The member descriptions should correspond to those in the member documentation.